surface/src/on_nurbs/on_nurbs.cmake

```diff
@@ -1,4 +1,24 @@
 
+message(STATUS "EIGEN_VERSION: ${EIGEN_VERSION}")
+message(STATUS "EIGEN_WORLD_VERSION: ${EIGEN_WORLD_VERSION}")
+message(STATUS "EIGEN_MAJOR_VERSION: ${EIGEN_MAJOR_VERSION}")
+message(STATUS "EIGEN_MINOR_VERSION: ${EIGEN_MINOR_VERSION}")
+
+#set(PREPROCESSOR_EIGEN_VERSION "-DEIGENVERSION_GEQUAL_3_2")
```

Comment:
Ok, why not. I don't think I have any really important stuff to commit. Most of it are minor fixes. Regarding NURBS fitting I intend to remove dependencies of PCL and just provide wrappers to make it usable in a more general way.